GUI/QtQuick/qml/Overview/AddressesTableView.qml

```diff
@@ -0,0 +1,144 @@
+
+/*
+
+***********************************************************************************
+* Copyright (C) 2018 - 2020, BlockSettle AB
+* Distributed under the GNU Affero General Public License (AGPL v3)
+* See LICENSE or http://www.gnu.org/licenses/agpl.html
+*
+**********************************************************************************
+
+*/
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+TableView {
+    id: component
+    width: 1200
+    height: 200
+
+    columnSpacing: 0
+    rowSpacing: 0
+    clip: true
+    boundsBehavior: Flickable.StopAtBounds
+
+    signal copyRequested(var id)
+
+    property bool has_copy: true
+
+    property color text_header_color: "#7A88B0"
+    property int text_header_size: 10
+
+    property color cell_text_color: "#FFFFFF"
+    property int cell_text_size: 11
+
+    property color separator_color: "#3C435A"
+    property color cell_background_color: "#333C435A"
+    property color selected_cell_background_color: "#22293B"
+    property color header_background_color: "#333C435A"
+
+    property var columnWidths: [350, 100, 120, 100]
```

Comment:
Not sure if column widths should be a fixed size. Better to have them as a percentage from the total TableView width for the case of table resizing with window.